src/app/core/models/left-nav-loading/nav-tree-data.ts

```diff
@@ -1,4 +1,5 @@
 import { ensureDefined } from 'src/app/shared/helpers/null-undefined-predicates';
```

Comment:
I think I found a new bug not related to this PR, but probably to the whole left-nav improvement thingy:
1. Go to [My groups](https://dev.algorea.org/en/#/groups/mine) directly (by pasting url, not by navigating inside the app)

:arrow_right: The left nav displays activities. Shouldn't it display `My groups` in the `Groups` tab ?

Reply:
Yes, I saw this yesterday as well. Unrelated with this PR, but we might want to fix it.

Reply:
Issue created. #749